modules/notification/action/action.go

```diff
@@ -402,10 +403,10 @@ func (a *actionNotifier) NotifySyncCreateRef(doer *user_model.User, repo *models
 	}
 }
 
-func (a *actionNotifier) NotifySyncDeleteRef(doer *user_model.User, repo *models.Repository, refType, refFullName string) {
+func (a *actionNotifier) NotifySyncDeleteRef(doer *user_model.User, repo *models.Repository, _, refFullName string) {
 	if err := models.NotifyWatchers(&models.Action{
-		ActUserID: repo.OwnerID,
-		ActUser:   repo.MustOwner(),
+		ActUserID: doer.ID,
+		ActUser:   doer,
```

Comment:
Is it a bug fix? If yes, we may need a separate PR.

Reply:
Not really - the passed argument `doer` is already the owner(in a lot of cases doer can be removed as it's present on things like issue and comments). So it's just "refactor" and use the already present parameter which already is the owner.

Reply:
I haven't read related code, but I just remember many bugs are caused by incorrect `owner`/`doer` usage (I also have used them wrongly before). In many cases, `doer` is not `owner`, for example, an admin user can do everything, then the doer should be the admin user, not the owner.

Reply:
> an admin user can do everything, then the doer should be the admin user, not the owner.

In this case, the only usage of the code is with `xxx.MustOwner()`(which I assume is the correct doer) passed into `doer`, but it would be a bug fix if the usage would pop somewhere else up whereby it's by a repo admin.